src/decode.rs

```diff
@@ -1452,7 +1452,7 @@ unsafe fn decode_b(
     }
 
     let ts = &mut *t.ts;
-    let f = &*t.f;
+    let f = &mut *t.f;
```

Comment:
Ask @rinon, but I think getting a `&mut Rav1dFrameContext` here might be difficult when we get rid of the raw pointer.  This is probably okay for now, though.